src/test/java/com/ncc/aif/ExamplesAndValidationTest.java

```diff
@@ -1765,8 +1770,8 @@ void setup() {
                 markJustification(markType(model, getAssertionUri(), relation,
                         SeedlingOntology.GeneralAffiliation_APORA, system, 1.0), justification);
 
-                //markImportance(makeClusterWithPrototype(model, getClusterUri(), event, system), 88);
-                markImportance(makeClusterWithPrototype(model, getClusterUri(), relation, system), 88);
+                //markImportance(makeClusterWithPrototype(model, getClusterUri(), event, system), 88.0);
```

Comment:
I'd suggest just deleting the commented out code, but I don't feel strongly about it.

Reply:
I just updated the importance values.  This commented code was there before and I seem to remember @sharknc or @ecurley-nextcentury had a reason for it.  So I'm leaving this here for now.